substrate/frame/revive/rpc/src/client/storage_api.rs

```diff
@@ -60,4 +65,44 @@ impl StorageApi {
 		let ContractInfo { trie_id, .. } = self.get_contract_info(address).await?;
 		Ok(trie_id.0)
 	}
+
+	/// Get the receipt data from storage.
+	pub async fn get_receipt_data(&self) -> Result<Vec<ReceiptGasInfo>, ClientError> {
+		let query = subxt_client::storage().revive().receipt_info_data();
+
+		let Some(receipt_info_data) = self.0.fetch(&query).await? else {
+			log::warn!(target: LOG_TARGET, "Receipt data not found");
```

Comment:
this might lead to too much logs for production usage, let's reduce all logs that could be triggered by a spammer to debug